phovea_registry.js

```diff
@@ -4,7 +4,7 @@
  * Licensed under the new BSD license, available at http://caleydo.org/license
  **************************************************************************** */
 
-import {register} from 'phovea_core/src/plugin';
+import {PluginRegistry} from 'phovea_core';
 import reg from './src/phovea';
```

Comment:
Is this correct now?

```suggestion
import reg from './dist/phovea';
```

Reply:
Yes, this should be correct.